modules/ROOT/pages/advanced-typography.adoc

```diff
@@ -0,0 +1,60 @@
+= Advanced Typography plugin
+:navtitle: Advanced Typography
+:description: Automatically or interactively replace typewriter-centric characters with characters preferred by typesetters.
+:description_short: Auomatically or interactively typeset {productname} documents.
```

Comment:
No automatic behaviour by default. Integrators would have to implement this themselves if they want such a feature.